src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { ErrorType } from '../types/ErrorType';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState<ErrorType>(ErrorType.NONE);
+
+  const showError = (error: ErrorType) => {
+    setIsError(error);
+    setTimeout(() => setIsError(ErrorType.NONE), 3000);
+  };
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        const personWithParents = data.map(person => {
+          const mother = data.find(mom => mom.name === person.motherName);
+          const father = data.find(dad => dad.name === person.fatherName);
+
+          return {
```

Comment:
Instead of using 'mom' and 'dad' as variable names, consider using more descriptive names like 'motherFound' and 'fatherFound' to improve code readability.